src/renderers/webgl/WebGLTextures.js

```diff
@@ -1433,74 +1459,37 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 	}
 
-
 	// Setup storage for internal depth/stencil buffers and bind to correct framebuffer
 	function setupRenderBufferStorage( renderbuffer, renderTarget, isMultisample ) {
 
 		_gl.bindRenderbuffer( _gl.RENDERBUFFER, renderbuffer );
 
-		if ( renderTarget.depthBuffer && ! renderTarget.stencilBuffer ) {
-
-			let glInternalFormat = _gl.DEPTH_COMPONENT24;
-
-			if ( isMultisample || useMultisampledRTT( renderTarget ) ) {
```

Comment:
I'm not sure why the if checks are different in the `renderTarget.depthBuffer && ! renderTarget.stencilBuffer` branch compared to `renderTarget.depthBuffer && renderTarget.stencilBuffer`. Why did you choose the below pattern?
```js
if ( isUseMultisampledRTT ) {

	multisampledRTTExt.renderbufferStorageMultisampleEXT( _gl.RENDERBUFFER, samples, glInternalFormat, renderTarget.width, renderTarget.height );

} else if ( isMultisample ) {

	_gl.renderbufferStorageMultisample( _gl.RENDERBUFFER, samples, glInternalFormat, renderTarget.width, renderTarget.height );

} else {

	_gl.renderbufferStorage( _gl.RENDERBUFFER, glInternalFormat, renderTarget.width, renderTarget.height );

}
```
and not this one:
```js
if ( isMultisample || useMultisampledRTT( renderTarget ) ) {

	if ( useMultisampledRTT( renderTarget ) ) {

		multisampledRTTExt.renderbufferStorageMultisampleEXT( _gl.RENDERBUFFER, samples, glInternalFormat, renderTarget.width, renderTarget.height );

	} else {

		_gl.renderbufferStorageMultisample( _gl.RENDERBUFFER, samples, glInternalFormat, renderTarget.width, renderTarget.height );

	}

} else {

	_gl.renderbufferStorage( _gl.RENDERBUFFER, glInternalFormat, renderTarget.width, renderTarget.height );

}
```
Are they functional equivalent?

Reply:
Yup! They're the same. I just thought it was easier to read. We can check using a table to make sure they get into the same branches.

*Condition Layout 1*

```js
if ( isUseMultisampledRTT ) {

	// Condition A

} else if ( isMultisample ) {

	// Condition B

}
```

*Table*

| | isUseMultisampledRTT = true | isUseMultisampledRTT = false |
|---|---|---|
| isMultisample = true | A | B |
| isMultisample = false | A | None |

*Condition Layout 2*

```js
if ( isMultisample || isUseMultisampledRTT ) {

	if ( isUseMultisampledRTT ) {

		// Condition A

	} else {

		// Condition B

	}

}
```

*Table*

| | isUseMultisampledRTT = true | isUseMultisampledRTT = false |
|---|---|---|
| isMultisample = true | A | B |
| isMultisample = false | A | None |

*Condition Layout 3*

```js
if ( isMultisample && isUseMultisampledRTT === false ) {

	// Condition B

} else if ( isUseMultisampledRTT ) {

	// Condition A

}
```

*Table*

| | isUseMultisampledRTT = true | isUseMultisampledRTT = false |
|---|---|---|
| isMultisample = true | A | B |
| isMultisample = false | A | None |

Reply:
Good! I definitely prefer the PRs approach.